test/integration/targets/nxos_netconf/tests/netconf/basic.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- debug: msg="START nxos_netconf cli/basic.yaml"
+
+- block:
+  - name: Configure vlan
+    netconf_config: &config_vlan
+      datastore: running
+      commit: false
+      get_filter: <System xmlns="http://cisco.com/ns/yang/cisco-nx-os-device"><bd-items><bd-items><BD-list></BD-list></bd-items></bd-items></System>
+      content: "{{ lookup('file', './configure_vlan.xml') }}"
```

Comment:
Better to add fixtures in a separate directory Refer: https://github.com/ansible/ansible/tree/devel/test/integration/targets/netconf_config/tests/junos/fixtures

and have it included in test file as a variable Refer https://github.com/ansible/ansible/blob/devel/test/integration/targets/netconf_config/tests/junos/basic.yaml#L4